test.js

```diff
@@ -316,11 +316,16 @@ test('non-deferred approximateSize', function (t) {
 })
 
 test('iterator - deferred operations', function (t) {
-  t.plan(8)
+  t.plan(9)
+
+  var hasSeeked = false
 
   var db = {
     iterator: function (options) {
       return {
+        seek: function () {
+          hasSeeked = true
```

Comment:
Should we do `t.is(target, ..)` here (and below) to assert the arguments are passed along?

Reply:
sure - done